src/api/cow/cow.spec.ts

```diff
@@ -215,6 +291,30 @@ test('Invalid: Get last 5 trades for an unexisting trader ', async () => {
   }
 })
 
+test('Valid: Get Price Quote from partial order', async () => {
+  fetchMock.mockResponseOnce(JSON.stringify(QUOTE_RESPONSE), { status: HTTP_STATUS_OK })
+  const quote = await cowSdk.cowApi.getQuote(QUOTE_REQUEST as FeeQuoteParams)
+  expect(fetchMock).toHaveBeenCalledTimes(1)
+  expect(fetchMock).toHaveBeenCalledWith('https://api.cow.fi/rinkeby/api/v1/quote', {
+    ...FETCH_RESPONSE_PARAMETERS,
+    body: JSON.stringify({
+      kind: 'buy',
+      buyAmountAfterFee: '1234567890',
+      sellToken: '0x6810e776880c02933d47db1b9fc05908e5386b96',
+      buyToken: '0x6810e776880c02933d47db1b9fc05908e5386b96',
+      from: '0x6810e776880c02933d47db1b9fc05908e5386b96',
+      receiver: '0x6810e776880c02933d47db1b9fc05908e5386b96',
+      appData: '0x0000000000000000000000000000000000000000000000000000000000000000',
+      validTo: 0,
+      partiallyFillable: false,
+    }),
```

Comment:
Shouldn't this bit of the body be equivalent to the string-fied QUOTE_RESPONSE constant?

At a bare minimum, could you use that to fill in the values here rather than repeating them